lib/utils.js

```diff
@@ -178,39 +179,20 @@ const parseConnectionIp = (sdp) => {
   return arr ? arr[1] : null;
 };
 
-/**
- * Converts IPv4 to an int
- * @param ipv4 IPv4 address, example 172.31.0.1
- * @returns an int representation of an ipv4 address
- * */
-const ip4ToInt = (ipv4) =>
-  ipv4.split('.').reduce((ipInt, octet) =>
-    (ipInt << 8) + parseInt(octet, 10), 0) >>> 0;
-
-/**
- * Checks IP against given CIDR
- * @param ip IPv4 address, example 172.31.0.1
- * @param cidr Array of IPv4 CIDR notations example ['172.31.0.0/16', '192.168.0.0/24']
- * @returns true if IP is present in network CIDR
- * */
-const isIp4InCidr = (ip) => (cidr) => {
-  const [range, bits = 32] = cidr.split('/');
-  const mask = ~(2 ** (32 - bits) - 1);
-  return (ip4ToInt(ip) & mask) === (ip4ToInt(range) & mask);
-};
-
 /**
  * Checks if ip is one of MS Teams sip signalling ips
  * https://learn.microsoft.com/en-us/azure/communication-services/concepts
  * /telephony/direct-routing-infrastructure#sip-signaling-fqdns
  * @param ip IP address, example 172.31.0.1
  * */
 const isMSTeamsCIDR = (ip) => {
+  const CIDRMatcher = require('cidr-matcher');
```

Comment:
could you move this to the top of the file, along with the other require's

Reply:
No problem, I did look at that originally then saw [Line 99 ](https://github.com/jambonz/sbc-inbound/blob/e5dce35babf9e9de44c73047caa209f44faf621c/lib/utils.js#L99C3-L99C38) and assumed that is how you wanted it. Let me know if you want me to move the express require at same time.

Reply:
hmm, the only time I usually do it like that is when I have to avoid a circular require, using when importing something that is a property of `srf`.  So I should probably at some point fix that other one as well

Reply:
yes if you could fix the other that would be great